typescript/pragma-scripts/README.md

```diff
@@ -18,14 +18,120 @@ bun install
 npm install
 ```
 
-# Available scripts
+Copy the `env.example` as `.env` and fill the variables.
 
-## update_feed
+## Available Scripts
 
-This script allows to update a data feed on a specific chain.
+### Dispatcher Scripts
 
-### Usage
+Located in the `dispatcher/` directory.
+
+#### add_feeds.ts
+
+Adds multiple data feeds to the Pragma Dispatcher contract.
+
+```bash
+bun run dispatcher/add_feeds.ts --chain <chain_name> --feed-ids <feed_id1> <feed_id2> ...
```

Comment:
update that to dispatcher:add_feeds as done in package.json

Reply:
done